draft-packages/illustration/KaizenDraft/Illustration/Base.tsx

```diff
@@ -0,0 +1,38 @@
+import { assetUrl } from "@kaizen/hosted-assets"
+import * as React from "react"
+const styles = require("./style.module.scss")
+
+export type BaseProps = {
+  /**
+   *  Refer to the Base Illustration Sticker Sheet in Zen UI Kit
+   */
+  name: string
+
+  /**
+   * If there is context/text surrounding this illustration that provides alt text,
+   * provide an empty string
+   */
```

Comment:
```suggestion
   * If there is context/text surrounding this illustration that serves
   * the function of alt text, provide an empty string so that any
   * accessibility checks know the alt text has been intentionally left out
   */
```

Reply:
Much clearer, thanks!